From ec2f3abf82b9a95ce2c2774a1b46d76943e24bec Mon Sep 17 00:00:00 2001
From: Everett Hildenbrandt <hildenb2@illinois.edu>
Date: Sat, 9 Sep 2017 21:10:14 -0500
Subject: [PATCH] build (new): adding script to generate build products and
 build

Note: We must recover the file `src/Mixfix/surface.cc` because it is not kept in
sync with the files which generate it.
---
 build | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100755 build

diff --git a/build b/build
new file mode 100755
index 0000000..732fad2
--- /dev/null
+++ b/build
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+automake --add-missing
+
+aclocal \
+    && autoconf \
+    && autoheader \
+    && automake \
+    && ./configure \
+    && make
+
+git checkout -- src/Mixfix/surface.cc
+
+make
-- 
2.14.1

